fastlane/Fastfile

```diff
@@ -0,0 +1,74 @@
+default_platform(:mac)
+
+platform :mac do
+	GITHUB_REMOTE = "origin"
+	RELEASE_BRANCH = "master"
+	GITHUB_REPOSITORY = "aaronraimist/seaglass"
+
+	VERSION = sh "bash ../scripts/version.sh"
+	ZIP_PATH = "/tmp/seaglass/upload/Seaglass-#{VERSION}.zip"
```

Comment:
Ran into problems here where `${VERSION}` had `/` in it, i.e. `pull/X`. I think I've fixed `scripts/version.sh` in `master` to hopefully resolve that.